packages/react-dom/src/server/ReactDOMFizzStaticNode.js

```diff
@@ -66,7 +66,7 @@ function prerenderToNodeStreams(
     const onFatalError = reject;
 
     function onAllReady() {
-      const readable = new Readable({
+      const readable: Readable = new Readable({
```

Comment:
question: I understand the type annotations for `let` variables where you might later widen the type. But it's not really clear why Flow needs the type annotation here. Is it because it doesn't understand the `Readable` constructor?

Reply:
I'm not quite sure and will raise that with the Flow team.

Reply:
The error is about the constructor containing a cyclic reference to itself. It seems like at least for non-generic types that shouldn't be required though.